pages/words/[id].js

```diff
@@ -22,24 +27,65 @@ export default function SingleEntry() {
             word: editedEntry.word,
             language: editedEntry.language,
             translated: editedEntry.translated,
+            notes: editedEntry.notes,
           };
         }
         return translation;
       })
     );
+    setIsShowMode(true);
   }
 
   return (
     <>
       <main>
-        <h1>Edit word</h1>
-        <CancelDeleteButton onClick={() => router.back()} />
-        {!entry ? null : (
-          <Form
-            entry={entry}
-            isEditMode={true}
-            onSubmitEvent={handleEditEntry}
-          />
+        {isShowMode ? (
+          <>
+            <h1>Word entry </h1>
+            <GoBackButton onClick={() => router.push("/words")} />
+            <EditButton onClick={() => setIsShowMode(false)}></EditButton>
+            {!entry ? null : (
+              <SingleEntry
+                isFavorite={entry.isFavorite}
+                word={entry.word}
+                language={entry.language}
+                translated={entry.translated}
+                notes={entry.notes}
+                onDeleteEntry={() => {
+                  router.push("/words");
+                  setTranslationList(
+                    translationList.filter(
+                      (translation) => translation.id !== id
+                    )
+                  );
+                }}
+                onToggleFavorite={() =>
+                  setTranslationList(
+                    translationList.map((translation) =>
+                      translation.id === id
+                        ? {
+                            ...translation,
+                            isFavorite: !translation.isFavorite,
+                          }
+                        : translation
+                    )
+                  )
+                }
+              ></SingleEntry>
```

Comment:
This can be self-closing:

```suggestion
              />
```

Reply:
Done